README.md

```diff
@@ -5,70 +5,102 @@ author: jmprieur
 ---
 
 # Integrating Azure AD into an ASP.NET Core web app
-This sample shows how to build a .Net MVC web application that uses OpenID Connect to sign-in users from a single Azure Active Directory tenant, using the ASP.Net Core OpenID Connect middleware.
 
-For more information about how the protocols work in this scenario and other scenarios, see [Authentication Scenarios for Azure AD](http://go.microsoft.com/fwlink/?LinkId=394414).
+This sample shows how to build a .NET MVC web app that uses OpenID Connect to sign-in users from a single Azure Active Directory tenant using the ASP.NET Core OpenID Connect middleware.
 
-> This sample has been updated to ASP.NET Core 2.0.  Looking for previous versions of this code sample? Check out the tags on the [releases](../../releases) GitHub page.
+For more information on how the protocols work in this scenario and other scenarios, see [Authentication Scenarios for Azure AD](http://go.microsoft.com/fwlink/?LinkId=394414).
 
-> If you are interested in  ASP.NET Core 2.1, please look at branch [aspnet_core_1_1](https://github.com/Azure-Samples/active-directory-dotnet-webapp-openidconnect-aspnetcore/tree/aspnet_core_1_1)
+## How to run this sample
 
-## How To Run This Sample
+To run this sample:
+- Install .NET Core for Windows by following the instructions at [.NET and C# - Get Started in 10 Minutes](https://www.microsoft.com/net/core). In addition to developing on Windows, you can develop on [Linux](https://www.microsoft.com/net/core#linuxredhat), [Mac](https://www.microsoft.com/net/core#macos), or [Docker](https://www.microsoft.com/net/core#dockercmd).
+- An Azure Active Directory (Azure AD) tenant. For more information on how to obtain an Azure AD tenant, see [How to get an Azure AD tenant](https://azure.microsoft.com/documentation/articles/active-directory-howto-tenant/).
 
-Getting started is simple!  To run this sample you will need:
-- Install .NET Core for Windows by following the instructions at [dot.net/core](https://dot.net/core), which will include Visual Studio 2015 Update 3.
-- An Internet connection
-- An Azure Active Directory (Azure AD) tenant. For more information on how to get an Azure AD tenant, please see [How to get an Azure AD tenant](https://azure.microsoft.com/en-us/documentation/articles/active-directory-howto-tenant/) 
-- A user account in your Azure AD tenant. This sample will not work with a Microsoft account, so if you signed in to the Azure portal with a Microsoft account and have never created a user account in your directory before, you need to do that now.
+### Step 1: Register the sample with your Azure Active Directory tenant
 
-### Step 1:  Clone or download this repository
+1. Sign in to the [Azure portal](https://portal.azure.com).
+2. Navigate to the Azure Active Directory blade.
```

Comment:
We need to explain a bit more here. people might have several tenant:
2. On the top bar, click on your account and under the **Directory** list, choose the Active Directory tenant where you wish to register your application
3. Click on **More Services** in the left hand nav, and choose **Azure Active Directory**.